stdlib/gc.mli

```diff
@@ -45,10 +45,24 @@ type stat =
 
     live_words : int;
     (** Number of words of live data in the major heap, including the header
-       words. *)
+       words.
+
+       Note that "live" words refers to every word in the major heap that isn't
+       currently known to be collectable, which includes words that haven't been
+       reachable by the program since after the start of the previous gc cycle.
```

Comment:
"since after" makes the sentence pretty hard to read. What about this:
```suggestion
       currently known to be collectable, which includes words that have become
       unreachable by the program after the start of the previous gc cycle.
```